Makefile.am

```diff
@@ -235,13 +268,92 @@ dist_bin_SCRIPTS += eos-updater-prepare-volume/eos-updater-prepare-volume
 
 dist_man8_MANS += eos-updater-prepare-volume/docs/eos-updater-prepare-volume.8
 
+# Flatpak installer utility library
+noinst_LTLIBRARIES += eos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUU_API_VERSION@.la
+
+eos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUU_API_VERSION@_la_CPPFLAGS = \
+	$(AM_CPPFLAGS) \
+	-DG_LOG_DOMAIN=\""eos-updater-flatpak"\" \
```

Comment:
s/`eos-updater-flatpak`/`eos-updater-flatpak-installer`/ so it matches the library name please.